app/controllers/tags_controller.rb

```diff
@@ -178,6 +178,8 @@ def list
   end
 
   def show
+    synonym_tag = Tag.where_name(params[:tag_id]).where.not(target_tag_id: nil).first
+    return redirect_to tag_show_path(synonym_tag.target_tag.name) if synonym_tag
```

Comment:
In `show_latest`, I see other params like `additional_tag_ids` and `periods`. If we just do a simple redirect here, I think those params will be lost?